src/apiTest/resources/com/ampnet/auditornode/script/api/classes/ipfs-test-properties.yaml

```diff
@@ -0,0 +1,2 @@
+ipfs:
+  gateway-url: http://localhost:8090/ipfs/{ipfsHash}
```

Comment:
To run `apiTest`, IPFS must be running locally?

Reply:
No really. During API tests, we are testing the full application without any service-level mocks (we cannot do that anyway since we also want to test agains the native image). So the only layer we can mock is HTTP API. If you look into the test which uses this property source (`DirectoryBasedIpfsTest` which I will rename to `DirectoryBasedIpfsApiTest`) you will see that it also starts WireMock server on port `8090` - this is returning mock responses to any IPFS requests that are made in the auditor application. We need to do this in order to test `DirectoryBasedIpfs` implementation via web socket API since there is no other way to mock it when testing agains the native image.